scenarios/aws/kindvm/run.go

```diff
@@ -154,3 +170,36 @@ agents:
 
 	return nil
 }
+
+func ConfigureECRCredentials(e resAws.Environment, vm *remote.Host, arch os.Architecture, opts ...pulumi.ResourceOption) (*goremote.Command, error) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
If this is meant to be used from `datadog-agent`, could be moved to `docker` or some helpers package, rather than scenarios.

Reply:
Since it is something specific to AWS I did not want to have it in `components/docker` that should remain cloud agnostic. But yes we can probably find a better place for it

Reply:
Maybe `resources/aws/ecr`